### ISSUE TITLE
adding missing data

### ISSUE THREAD

**Issue by BurntSushi** [BurntSushi/nflgame#63]
@ochawkeye brought this up in #62:

> That is unfortunate as it does look like that entire last 14 minutes and 56 seconds of the 4th quarter are missing from this game. For guys like Antonio Brown those are some significant stats that are missing.
> 
> In my own script, I have added a section to manually include some of the "missing" data and/or stat corrections that come down from the NFL mid-week. Is there a method for doing this sort of thing for nflgame? Obviously these manual tweaks aren't ideal, but with the community that has been growing I imagine we are getting close to the numbers necessary to make it easier to crowd source this than to have multiple people doing it on their own.
> 
> I know it hasn't been your highest priority goal to aspire to make nflgame 100% accurate, but this would allow to get closer to true and eliminate some of those gaps that accumulate over the course of a year as well.

**Comment by jaunt7** [comment]
This guy tried to parse the grammar a bit back - http://www.10flow.com/2013/01/10/nfl-play-by-play-data/
Parser at https://github.com/10flow/playbyplay

**Comment by jaunt7** [comment]
For completely missing plays, it might be sufficient to cross-reference the human-readable source for missing times.  If a time is recorded that doesn't exist in the plays, then clearly we are missing plays.
Have you seen this to continue to be a big problem since the big missing end of game in reference for this thread?  
As for the game I referenced in #184, the play had yards net of 28, but it didn't credit it to the player (stat had zero yards) - maybe we could flag games as "needing to be manually looked at" if the yards net is greater than zero but nobody was assigned yards, for certain stat numbers, or ignoring certain descriptions. 
Over the years have you accumulated a list of the common errors for the manual fixes you employ to run your league?

**Comment by BurntSushi** [comment]
@jaunt7 That sounds like a fine idea. There is almost no end to the number of heuristics one might adopt to suss out bad data. I personally probably won't lead that task, although I might be happy to maintain it if it interoperates with nfldb nicely.

**Comment by jaunt7** [comment]
Of course, you've done enough work on your own.  I will let you know if I come up with something worth contributing.
